test/others/test_schema.py

```diff
@@ -144,6 +201,25 @@ def test_label_to_dict(text_labels):
     assert l_new.answer.offsets_in_document[0].start == 1
 
 
+def test_equal_table_label(table_label_1, table_label_2):
+    assert table_label_1 == table_label_1
+    assert table_label_1 != table_label_2
+
+
+def test_table_label_to_json(table_label_1):
+    j0 = table_label_1.to_json()
+    l_new = Label.from_json(j0)
```

Comment:
I feel this is a bit dangerous. It certainly makes sense to compare the from/to conversion but I'd prefer if we do it from an hardcoded json and we always load that.

If we introduce a bug in `to_json` at the same time we introduce it in `from_json` we won't notice otherwise.

Storing that json in a separate file is good for me.

Reply:
haha yup for sure, again most of this is copying previous format in this file. I'd be happy to go through and improve the tests throughout this file.